### PR TITLE
Pointer fix

### LINKED ISSUE #639: Change cursor to pointer
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [x] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [x] 📝 **Update**

Selection_162

**Changes**: Chance cursor to pointer download and pan to this image
Make changes in file app/views/images/_index.html.erb Change 
https://github.com/publiclab/mapknitter/blob/dc96adae1b376e7a90a3410c535dd547230caf5e/app/views/images/_index.html.erb#L42
https://github.com/publiclab/mapknitter/blob/dc96adae1b376e7a90a3410c535dd547230caf5e/app/views/images/_index.html.erb#L43
- [x] 💾 **Commit** your changes

- [x] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [x] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by metonymic-smokey**
Fixes #639 

Changed 2 files _index.html.erb and _new.html.erb
Also, changed mapknitter.env to have rails environment set to development(since these changes were made locally while in dev mode).

Thanks!

### CODE REVIEW

File: app/views/images/_index.html.erb [hunk 1]
```diff
@@ -83,11 +83,16 @@
             %>
           <% end %>
           <% if warpable.placed? %>
+<<<<<<< HEAD
```
Comment (divyabaid16) [inline app/views/images/_index.html.erb]:
```suggestion
```

File: app/views/images/_index.html.erb [hunk 2]
```diff
@@ -83,11 +83,16 @@
...
+            <a rel="tooltip" id="locate-warpable-<%= warpable.id %>" title="Pan to this image" style="cursor: pointer" class="btn btn-sm btn-success" style="word-wrap: break-word"><i class='fa fa-crosshairs'></i></a>
+            <a rel="tooltip" title="Download full resolution distorted image (please allow popups)" style="cursor: pointer" class="btn btn-sm btn-outline-secondary img-download img-download-<%= warpable.id %>" data-image="/images/fetch/?url=<%= warpable.image.url(:original) %>"style="word-wrap: break-word"><i class="fa fa-download"></i></a>
+=======
```
Comment (divyabaid16) [inline app/views/images/_index.html.erb]:
```suggestion
```

File: app/views/images/_index.html.erb [hunk 3]
```diff
@@ -83,11 +83,16 @@
...
             <a id="locate-warpable-<%= warpable.id %>" title="Pan to this image" class="btn btn-sm btn-success"><i class='fa fa-crosshairs'></i></a>
             <a title="Download full resolution distorted image (please allow popups)" class="btn btn-sm btn-outline-secondary img-download img-download-<%= warpable.id %>" data-image="/images/fetch/?url=<%= warpable.image.url(:original) %>"><i class="fa fa-download"></i></a>
             <a title="History of the image" class = "btn btn-sm btn-warning" onClick="$('#image-vesrions-<%= warpable.id %>').toggle()" style="cursor: pointer;">
             <i class='fas fa-clock'></i>
             </a>
+>>>>>>> upstream/main
```
Comment (divyabaid16) [inline app/views/images/_index.html.erb]:
```suggestion
```

File: app/views/images/_index.html.erb [hunk 4]
```diff
@@ -140,6 +145,8 @@
     <% end %>
   </tbody>
 </table>
+<<<<<<< HEAD
```
Comment (divyabaid16) [inline app/views/images/_index.html.erb]:
```suggestion
```

File: app/views/images/_index.html.erb [hunk 5]
```diff
@@ -150,4 +157,5 @@
   function list(){
     $('td').show()
   }
-</script>
\ No newline at end of file
+</script>
+>>>>>>> upstream/main
```
Comment (divyabaid16) [inline app/views/images/_index.html.erb]:
```suggestion
```

**Comment by metonymic-smokey** [comment]
Thanks @divyabaid16 ! 
So sorry for the merge conflicts. Still new to this so sometimes end up with these, figured this out for upcoming PRs now!

File: app/views/images/_index.html.erb [hunk 6]
```diff
@@ -140,6 +142,7 @@
     <% end %>
   </tbody>
 </table>
+=======
```
Comment (divyabaid16) [inline app/views/images/_index.html.erb]:
```suggestion
```
Just remove this and then this can be merged.
Thanks!

**Comment by metonymic-smokey** [comment]
@divyabaid16 anything else to be done for this PR so that it can pass the codecov test?

Thanks!

**Comment by divyabaid16** [comment]
It's great. I guess it has some merge conflict. Can you please rebase your PR?
Thanks!

**Comment by divyabaid16** [comment]
@gauravano @jywarren

**Comment by metonymic-smokey** [comment]
Thanks a lot @divyabaid16 !

File: app/views/images/_index.html.erb [hunk 7]
```diff
@@ -83,6 +83,8 @@
             %>
           <% end %>
           <% if warpable.placed? %>
+            <a rel="tooltip" id="locate-warpable-<%= warpable.id %>" title="Pan to this image" style="cursor: pointer" class="btn btn-sm btn-success" style="word-wrap: break-word"><i class='fa fa-crosshairs'></i></a>
+            <a rel="tooltip" title="Download full resolution distorted image (please allow popups)" style="cursor: pointer" class="btn btn-sm btn-outline-secondary img-download img-download-<%= warpable.id %>" data-image="/images/fetch/?url=<%= warpable.image.url(:original) %>"style="word-wrap: break-word"><i class="fa fa-download"></i></a>
```
Comment (gauravano) [inline app/views/images/_index.html.erb]: I think, the original issue intend to change the cursor to pointer on existing button. You have to make changes to existing links.

Comment (metonymic-smokey) [reply app/views/images/_index.html.erb]: @gauravano this PR does change to a pointer on hovering over the buttons(screenshots attached in issue).
Sorry I'm not entirely clear on what you mean. Could you tell me what you mean by 'making changes to existing links'?

Thanks!

Comment (gauravano) [reply app/views/images/_index.html.erb]: The links are already present so you just have to make changes. See the next two lines where you added new lines and compare. Aren't they same??

**Pull request merged**